music21/romanText/translate.py

```diff
@@ -1605,6 +1606,94 @@ def testTuplets(self):
         self.assertEqual(n2.offset, common.opFrac(11 / 6))
         self.assertEqual(n2.duration.quarterLength, common.opFrac(13 / 6))
 
+    def testRepeats(self) -> None:
+        from music21 import converter
+
+        def _repeat_tester(
+            repeat_bar: bar.Repeat, direction: str, offset: float, mNumber: int
+        ) -> None:
+            self.assertEqual(repeat_bar.direction, direction)
+            self.assertEqual(repeat_bar.offset, offset)
+            self.assertEqual(repeat_bar.activeSite.number, mNumber)
+
+        def _test_expanded(s: stream.Stream, quarterLength: float) -> None:
+            # NB repeat.Expander does not work on stream, only on part. Is this
+            #   expected behavior?
+            p = next(s[stream.Part])
+            e = repeat.Expander(p)
+            self.assertTrue(e.repeatBarsAreCoherent())
+            p2 = e.process()
+            self.assertEqual(p2.quarterLength, quarterLength)
+
+        # Test simple repeats
+        simple_repeats = '''Time Signature: 3/4
+m1: ||: V
+m2: I :||
+'''
+        s = converter.parse(simple_repeats, format='romanText')
+        start_repeat, end_repeat = s[bar.Repeat]
```

Comment:
better to assign to an iterator and test that `self.assertEqual(len(iterator), 2)` so that if something fails later we can see what the problem is rather than a cryptic "to few/many elements to unpack".  Again, NBD.